json-schemas/interface-methods/protocol-rule-set.json

```diff
@@ -106,10 +106,54 @@
           "minimum": 0
         }
       }
+    },
+    "$tags": {
+      "type": "object",
+      "minProperties": 1,
+      "patternProperties": {
+        "requiredTags": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "allowUndefinedTags": {
+          "type": "boolean"
+        },
```

Comment:
Sanity: it's odd that these two properties are placed within `patternProperties` given that they are predefined. Is it possible to move them out:

```json
      "properties": {
        "requiredTags": {
            "type": "array",
            "items": {
                "type": "string"
            }
        },
        "allowUndefinedTags": {
            "type": "boolean"
        }
      },
```

Reply:
Seems to be okay: https://github.com/TBD54566975/dwn-sdk-js/pull/722

Reply:
🙏  Thanks!! Yeah that is much better (and probably more performant) great catch!